images/cassandra/configs/latest.apko.yaml

```diff
@@ -0,0 +1,50 @@
+contents:
+  packages:
+    - cassandra
+    - python3
+    - bash 
```

Comment:
shouldnt this only be added for dev? same with busybox?

Reply:
I think python3 is needed by [cqlsh]( https://cassandra.apache.org/doc/latest/cassandra/tools/cqlsh.html ) and even bash/busybox was required as much I know since it was needed by Cassandra (which is basically a script in opt/cassandra/bin/ and it wasn't building/running without that but @ajayk can provide all the more context 🤔

Reply:
Just checked again, it can work without bash so removing that but it is failing if we remove busybox/python3

Reply:
cc @rawlingsj